README.md

```diff
@@ -159,7 +159,10 @@ summary_num_results: 10
 summary_prompt_name: vectara-summary-ext-v1.2.0
 
 # Whether to enable the Hughes Hallucination Evaluation Model (HEM - https://github.com/vectara/hallucination-leaderboard)
```

Comment:
Nit pick: suggest we rephrase the comment to "Whether to enable the HHEMv2 (based on https://huggingface.co/vectara/hallucination_evaluation_model), also known as factual consistency score"